### DISCUSSION TITLE
Change default theme from lupin (current) to dark

### DISCUSSION THREAD

**Discussion by AnonymousWP** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#123] [Polls]
Context: https://github.com/Universal-Debloater-Alliance/universal-android-debloater/issues/122

**Comment by ghost** [comment]
honestly light theme is better, dark theme sucks  

there is no point in favor of dark mode , except "save battery"  
but is saving that battery worth the eye strain?  

heck its not even aesthetically pleasing.

**Reply by ghost** [reply]
https://youtu.be/12FP8vF0eNs

**Reply by AnonymousWP** [reply]
I know that using dark theme can cause eye-strain, which is why I personally always use light mode in every application I can during daytime. However, in a dark room at night I prefer a dark theme. Follow system theme would be better, or you know, having a choice is even better. Makes everyone happy. 🙂

**Reply by ghost** [reply]
Night is for 😴 sleeping (pls don't ignore your health). 

A reason why it's called "dark theme" and not "night theme" ( I didn't notice it till now)🤯. 😂

**Reply by AnonymousWP** [reply]
You're taking 'night' too literal. It also applies to the evening, of course. And it's everyone's own decision what time he or she is going to sleep.

**Comment by AnonymousWP** [comment]
Done in https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/releases/tag/v1.0.0.
